botorch/acquisition/joint_entropy_search.py

```diff
@@ -12,44 +12,76 @@
 
 from __future__ import annotations
 
+import warnings
+from math import log, pi
+
 from typing import Any, Optional
 
-from botorch.acquisition.multi_objective.joint_entropy_search import (
-    qLowerBoundMultiObjectiveJointEntropySearch,
-)
-from botorch.acquisition.multi_objective.utils import compute_sample_box_decomposition
+import torch
+from botorch import settings
+from botorch.acquisition.acquisition import AcquisitionFunction, MCSamplerMixin
+from botorch.acquisition.objective import PosteriorTransform
+
+from botorch.models.fully_bayesian import SaasFullyBayesianSingleTaskGP
+from botorch.models.gp_regression import MIN_INFERRED_NOISE_LEVEL
 from botorch.models.model import Model
+
+from botorch.models.utils import check_no_nans, fantasize as fantasize_flag
+from botorch.sampling.normal import SobolQMCNormalSampler
 from botorch.utils.transforms import concatenate_pending_points, t_batch_mode_transform
 from torch import Tensor
 
+from torch.distributions import Normal
+
+MCMC_DIM = -3  # Only relevant if you do Fully Bayesian GPs.
+ESTIMATION_TYPES = ["MC", "LB"]
+MC_ADD_TERM = 0.5 * (1 + log(2 * pi))
 
-class qLowerBoundJointEntropySearch(qLowerBoundMultiObjectiveJointEntropySearch):
+# The CDF query cannot be strictly zero in the division
+# and this clamping helps assure that it is always positive.
+CLAMP_LB = torch.finfo(torch.float32).eps
+FULLY_BAYESIAN_ERROR_MSG = (
+    "JES is not yet available with Fully Bayesian GPs. Track the issue, "
+    "which regards conditioning on a number of optima on a collection "
+    "of models, in detail at https://github.com/pytorch/botorch/issues/1680"
+)
+
+"""
+References
+.. [Hvarfner2022joint]
+    C. Hvarfner, F. Hutter, L. Nardi,
+    Joint Entropy Search for Maximally-informed Bayesian Optimization.
+    In Proceedings of the Annual Conference on Neural Information
+    Processing Systems (NeurIPS), 2022.
+
+.. [Tu2022joint]
+    B. Tu, A. Gandy, N. Kantas, B. Shafei,
+    Joint Entropy Search for Multi-objective Bayesian Optimization.
+    In Proceedings of the Annual Conference on Neural Information
+    Processing Systems (NeurIPS), 2022.
+"""
```

Comment:
This is what trips sphinx up. Can you just move these references in the module-level docstring at the top (l.7) instead?